src/content/roadmap/_index.en.md

```diff
@@ -25,14 +25,17 @@ Please see details of previous releases [here](../releases)
 
 ### v0.3.0
 
+* Auto detecting NAT vs non-NAT scenarios.
+* Supporting different ports for IPSEC for each cluster
 * Removing the kubefed dependency from Lighthouse service discovery
 * Measuring and improving A/P HA (different scenarios)
 * Libreswan support
 
 ### v0.4.0 
 
 * Support for network policies via coastguard
-* Monitoring and reporting of tunnel endpoints
+* Monitoring and reporting of tunnel endpoints (status of connection, bandwidth, pps, etc..)
+* Monitoring connectivity over port 4800 between routeagent nodes.
```

Comment:
nit: Monitoring VxLAN UDP (port 4800) connectivity between routeagent nodes.

Reply:
It’s just a roadmap :-P :-) I’ll merge it to move forward |-)